tests/src/test/resources/com/powsybl/openrao/tests/features/epic10_relative_margin/US10_1.feature

```diff
@@ -5,21 +5,21 @@
 
 Feature: US 10.1: Linear RAO with relative margin
 
-  @fast @rao @mock @ac @preventive-only @relative
+  @fast @rao @mock @dc @preventive-only @relative
```

Comment:
Is there a reason you moved from AC to DC? I am affraid this will change the LF results which could alter the test's results

Reply:
The same question goes for all the other tests, I would rather convert by hand the objective value from MW to A instead of changing the LF computation mode

what do you think @phiedw?

Reply:
This dev will be replaced by a separate unit parameter in rao parameter, then no change in tests behavior is expected.